### ISSUE TITLE
tf.keras.model.evaluate() not working as expected

### ISSUE THREAD

**Issue by warriorgiggles** [tensorflow/tensorflow#41291]
**System information**
Code is running locally on:

MacBook Pro
2.8 GHz Quad-Core Intel Core i7
Radeon Pro 555 2 GB
Intel HD Graphics 630 1536 MB

**current behavior**

In Jupyter

`
import tensorflow as tf

mnist = tf.keras.datasets.fashion_mnist
(training_images, training_labels), (test_images, test_labels) = mnist.load_data()
training_images = training_images / 255.0
test_images = test_images / 255.0
model = tf.keras.models.Sequential([
    tf.keras.layers.Flatten(),
    tf.keras.layers.Dense(128, activation=tf.nn.relu),
    tf.keras.layers.Dense(10, activation=tf.nn.softmax)
])
model.compile(optimizer='adam', loss='sparse_categorical_crossentropy', metrics=['accuracy'])
model.fit(training_images, training_labels, epochs=5)

test_loss = model.evaluate(test_images, test_labels)
`

The output for the evaluate method  is currently "10000/1" followed by thousands of "==". I also believe the fraction should be "10000/10000"

**expected behavior**

something like: "10000/10000 [==========]"

** code to reproduce**

pasted above, exactly as is in a jupyter notebook

**Comment by warriorgiggles** [comment]
some more details:

tf.__version__

provides the following:

2.0.0-rc1


but pip list shows:

tensorflow             2.2.0

**Comment by warriorgiggles** [comment]
fixed the issue - something to do with the jupyter notebook using the wrong kernel
